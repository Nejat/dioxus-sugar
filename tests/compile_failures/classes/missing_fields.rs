use dioxus_sugar::classes;

#[classes(id)]
struct Props {
    name: String,
}

fn main() {}