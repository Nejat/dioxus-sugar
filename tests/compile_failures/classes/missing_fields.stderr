error[E0609]: no field `id` on type `&Props`
 --> tests/compile_failures/classes/missing_fields.rs:3:11
  |
3 | #[classes(id)]
  |           ^^ unknown field
  |
  = note: available fields are: `name`
